python-checks/src/test/java/org/sonar/python/checks/UnusedLocalVariableCheckTest.java

```diff
@@ -35,4 +36,19 @@ public void custom() {
     check.format = "(_|myignore)";
     PythonCheckVerifier.verify("src/test/resources/checks/unusedLocalVariableCustom.py", check);
   }
+
+  @Test
+  public void quickFixTest() {
```

Comment:
Also, there is another edge case with the quick fix, where it slightly changes the semantics of the program:
```python
def using_tuples():
    x, y = (1, 2)
    y = 5
    print(x)
```
Replacing `y` with `_` is not erroneous, but it somewhat changes program semantics. I think we can document this case with another test.

Reply:
done